tv.sh

```diff
@@ -8,34 +8,38 @@ TITLES=(
   "SIC"
   "TVI"
   "RTP 3"
-  "SIC Noticias"
+  "SIC Notícias"
   "CNN Portugal"
-  "RTP Memoria"
+  "RTP Memória"
   "RTP Madeira"
-  "RTP Acores"
-  "RTP Africa"
+  "RTP Açores"
+  "RTP África"
   "RTP Internacional"
-  "SIC Internacional"
+  "TVI Internacional"
+  "TVI Ficção"
+  "TVI Reality"
   "ARTV"
   "Porto Canal"
   "Euronews"
   "Kuriakos TV"
 )
 
 STREAMS=(
-  "__rtp rtp1"
-  "__rtp rtp2"
-  "https://sic.live.impresa.pt/sic.m3u8"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtp1HD.smil/rtp_playlist.m3u8"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtp2HD.smil/rtp_playlist.m3u8"
+  "https://d1zx6l1dn8vaj5.cloudfront.net/out/v1/b89cc37caa6d418eb423cf092a2ef970/index.m3u8"
```

Comment:
Is this the official stream?

Reply:
BTW, the existing SIC stream was working for me (from Portugal, without VPN).
I don't consider that it requires an update.

Reply:
Yes, it's official I have added it to the [M3UPT](https://github.com/LITUATUI/M3UPT) list long ago. 

The only difference is that this one has no geoblocking, they are both from SIC servers on cloudfront.net.

Reply:
Furthermore, the stream without geoblock has less delay, you'll notice it when zapping.